onadata/libs/utils/export_tools.py

```diff
@@ -163,8 +163,10 @@ def generate_export(export_type, xform, export_id=None, options=None,
     if options.get("dataview_pk"):
         dataview = DataView.objects.get(pk=options.get("dataview_pk"))
         records = dataview.query_data(dataview, all_data=True)
+        total_records = dataview.query_data(dataview, count=True)
     else:
         records = query_data(xform, query=filter_query, start=start, end=end)
+        total_records = xform.num_of_submissions
```

Comment:
Does this take into account the filter query? An export could be using filters where the number of records is less than the total number of submissions.